model/pipeline/src/main/proto/external_transforms.proto

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * Protocol Buffers describing the external transforms available.
+ */
+
+syntax = "proto3";
+
+package org.apache.beam.model.pipeline.v1;
+
+option go_package = "pipeline_v1";
+option java_package = "org.apache.beam.model.pipeline.v1";
+option java_outer_classname = "ExternalTransforms";
```

Comment:
From a Python SDK perspective, these may be "external" transforms. But in general that depends on which SDK and expansion service / runner is used. Conceivably the same could be implemented in Py, Go or any other language. Maybe "Default Transforms" or "Standard Transforms" would be more appropriate?

Reply:
The name can be refined but we would only use these messages for SDKs which are not written in the transform's language. If the language is the same, we do not have to use the `ExpansionService` to send an `ExpansionRequest`. We can just directly instantiate it.